Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -180,3 +180,29 @@ def customiseRun2CommonHI(process):
 
     return process
 
+##############################################################################
+# modifications of muon DQMOffline for pPb run
+def customiseRun2PPB_MuonDQMOffline(process):
```

Comment:
if these modifications apply to MC as well, please move them to a relevant DQM package.
Edits in this file can be relevant only for data.